log/context.go

```diff
@@ -25,6 +27,8 @@ type (
 
 // WithLogger returns a new context with the provided logger. Use in
 // combination with logger.WithField(s) for great effect.
+// If you use WithLogger, make sure the logger you're adding has a module field
```

Comment:
Can you put a check for this in the method ?

Reply:
That's kind of expensive to do for every call to `WithLogger`. `module` isn't actually required, so this comment is incorrect.